### PR TITLE
fix(back and front): fix bug of build 128

### PR DESCRIPTION
**Pull request by floreal15**
https://github.com/tracim/tracim/issues/6436

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

File: backend/tracim_backend/command/periodic.py [hunk 1]
```diff
@@ -58,9 +63,9 @@ def _send_mail(
         reply_to_address = config.EMAIL__NOTIFICATION__FROM__EMAIL.replace("{user_id}", "0")
 
         msg = EmailNotificationMessage(
-            subject=_("[{website_title}] Your daily summary").replace(
-                EST.WEBSITE_TITLE, config.WEBSITE__TITLE.__str__()
-            ),
+            subject=_("[{website_title}] Your {email_notification_type} summary")
+            .replace(EST.WEBSITE_TITLE, config.WEBSITE__TITLE.__str__())
+            .replace("{email_notification_type}", email_notification_type.__str__()),
```
Comment (Skylsmoi) [inline backend/tracim_backend/command/periodic.py:68]: you can use f-string for string interpolation

Not required change

File: backend/tracim_backend/models/data.py [hunk 2]
```diff
@@ -106,6 +106,18 @@ def to_string(self, lang, translator) -> Optional[str]:
...
+        if self == self.INDIVIDUAL:
+            return self.value
+        if self == self.NONE:
+            return self.value
+        if self == self.HOURLY:
+            return translator.get_translation("hourly", lang)
+        if self == self.DAILY:
+            return translator.get_translation("daily", lang)
+        if self == self.WEEKLY:
+            return translator.get_translation("weekly", lang)
```
Comment (Skylsmoi) [inline backend/tracim_backend/models/data.py:119]: Are these keys translated?
I don't see them added to translation file

**Review by Skylsmoi — APPROVED** [review]

File: backend/tracim_backend/models/data.py (hunk 2 above)
Comment (floreal15) [reply backend/tracim_backend/models/data.py:119]: The translation isn't done yet

**Review by Skylsmoi — APPROVED** [review]

**Comment by Skylsmoi** [comment]
end to end tests failed because of unstable test

**Pull request merged**